### DISCUSSION TITLE
How is it going with the new Figma UI Kit (for PV4)?

### DISCUSSION THREAD

**Discussion by d49017** [primefaces/community#2835] [Figma UI Kit] [answered]
Hi, according to the roadmap on the primevue website phase 2 of the figma ui kit is half-way complete (and was supposed to be finished in q3?). 

Will you release the new ui kit compatible with the primevue 4 when the phase 2 is done, or will you wait until the figma to theme api is also finished?

**Comment by onursenture** [comment, marked as answer]
A new version (v3) of the UI Kit was released last week. It is compatible with the new theming used in PrimeVue 4.
